### DISCUSSION TITLE
Recommended model, CUDA, drivers, etc. for a 4070 TI 16 VRAM❓

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1128] [Q&A] [answered]
I just got a **4070 Ti 16Gb Super Asus Tuf Gaming Oc** (i come from a 1080 TI) and i really don't know if i should wipe everything (CUDA, drivers, etc..) and get the last one of everything...

**PD:** RAM 32gb.

**Comment by wizz13150** [comment, marked as answer]
My go to with my 3060 - 12gb regarding the driver is v546.01
Everything after this version is a gain for xx90 series, but a speed loss for me. (£#%$!)
Give it a try. (Include the No Fallback Sysmem Policy)

**Reply by ZeroCool22** [reply]
> My go to with my 3060 - 12gb regarding the driver is v546.01 Everything after this version is a gain for xx90 series, but a speed loss for me. (£#%$!) Give it a try. (Include the No Fallback Sysmem Policy)

I remember some time ago that option for Nvidia control panel: **(Include the No Fallback System Policy)** was slowing down generation speeds on SDXL, so now it's safe to use or we only need it for FLUX only?

Also, what option should i use?

![Screenshot_3](https://github.com/user-attachments/assets/04ee8b15-fedc-4046-aceb-4dee41f5a3b4)

I have a 5900x

And **VAE/ENCODER** is empty:

![Screenshot_4](https://github.com/user-attachments/assets/65678bf9-a784-411b-b478-6631b963ddaa)

**Reply by wizz13150** [reply]
'Shared' for Swap Location is supposed to eventually be faster. I didn't notice any change with this one, tho.

Regarding the 'No Sysmem Fallback Policy', i'm always using it. Therefore I have to be careful with the resolutions. 
My max is about 2.2MPixels per tile/image, then OOM. (I'm still using A1111 API until the Forge API is done. Waiting to finally switch to Forge)

VAE/TE are included in the NF4 versions. You'll need it if using another model.

Ah, btw, use the Studio driver, so you can see the CUDA usage in the task manager.
![image](https://github.com/user-attachments/assets/14cf7da4-ad41-45e3-bee9-6bbfb3535e9e)

Cheers ! 🥂
